### PR TITLE
feature/sc-132/backend-module-for-events

### PR DESCRIPTION
**Pull request by oystebha**
-feat(events): add model, schema and resolvers for Event, add to admin.

### CODE REVIEW

File: events/models.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    class Meta:
+        default_related_name = "events"
+        verbose_name_plural = "Events"
+
+    title = models.CharField(unique=True, max_length=80)
+    description = models.CharField(max_length=1024)
+    date = models.DateField(blank=True, null=True)
+
+    max_participants = models.IntegerField(default=10)
+    participants = models.ManyToManyField(User, blank=True)
```
Comment (alexaor) [inline events/models.py]: Think we might make the reverse relation a bit more clearer with a more explicit `related_name`, maybe something like this?
```suggestion
    participants = models.ManyToManyField(User, blank=True), related_name="attended_events"
```

File: events/schema.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    def get_node(cls, info, id):
+        return Event.objects.get(pk=id)
+
+
+class EventQuery(graphene.ObjectType):
+    event = Node.Field(EventNode)
+    all_events = DjangoConnectionField(EventNode)
+
+    def resolve_all_events(self, info, *args, **kwargs):
+        return Event.objects.all()
```
Comment (alexaor) [inline events/schema.py]:
```suggestion
        return Event.objects.all().order_by("-date")
```

File: events/models.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class Event(models.Model):
+    class Meta:
+        default_related_name = "events"
+        verbose_name_plural = "Events"
+
+    title = models.CharField(unique=True, max_length=80)
+    description = models.CharField(max_length=1024)
+    date = models.DateField(blank=True, null=True)
```
Comment (alexaor) [inline events/models.py]: Is it better if this is a DateTime field? Then we could implicitly define the meeting time as well

**Review by alexaor — CHANGES_REQUESTED** [review]
Good stuff, some minor model/fields comments

File: events/models.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+from organization.models import InternalGroupPosition
+
+
+class Event(models.Model):
+    class Meta:
+        default_related_name = "events"
+        verbose_name_plural = "Events"
+
+    title = models.CharField(unique=True, max_length=80)
+    description = models.CharField(max_length=1024)
```
Comment (alexaor) [inline events/models.py]: Make this a `TextField` instead of a `CharField`?

**Comment by oystebha** [comment]
done @ all

**Review by alexaor — APPROVED** [review]
LGTM

**Pull request merged**